components/ButtonGenerator/index.tsx

```diff
@@ -379,6 +383,8 @@ export default function ButtonGenerator (): JSX.Element {
                   randomSatoshis={button.randomSatoshis}
                   hideToasts={button.hideToasts}
                   disableEnforceFocus={button.disableEnforceFocus}
+                  contributionOffset={button.contributionOffset}
+                  disableAltpayment={button.disableAltpayment}
```

Comment:
same here. lets make sure we're adding all the new props here so the preview button has the intended functionality. Its unlikely anyone would actually test a different Api base url here, or the auto close, but should add it anyway